composer.json

```diff
@@ -240,7 +240,8 @@
             },
             "drupal/group": {
                 "2842630 - Empty page when trying to create group node": "https://www.drupal.org/files/issues/2019-01-04/2842630-20.patch",
-                "3020883 - Use VBO together with group permission": "https://www.drupal.org/files/issues/2020-08-17/3020883-22.patch"
+                "3020883 - Use VBO together with group permission": "https://www.drupal.org/files/issues/2020-08-17/3020883-22.patch",
+                "3268711 - Error: Call to a member function label() on null in line 131 of modules/contrib/group/src/Entity/Controller/GroupListBuilder.php).": "https://www.drupal.org/files/issues/2022-03-09/Group-3268711-1.patch"
```

Comment:
I'm not seeing the error this patch refers to on any environment. The issue states that the patch is for Drupal 9.3 and Groups 1.4. We are using Drupal 9.2 still on portlandor. Please remove this patch or explain its use.

Reply:
The error was quite similar to the one on the issue, and was produced on my local: 
`Error: Call to a member function label() on null in Drupal\group\Entity\Controller\GroupListBuilder->buildRow() (line 131 of modules/contrib/group/src/Entity/Controller/GroupListBuilder.php).`

How would I go about producing this error on an environment? Do you mean producing it on a multidev?

Reply:
Good news, the fresh branch didn't need the patch